plone/app/z3cform/templates/orderedselect_input.pt

```diff
@@ -104,6 +105,7 @@
                   onchange view/onchange;
                   multiple view/multiple;
                   size view/size;
+                  required required | nothing;
```

Comment:
I'd follow best practice of the other widget templates and use a onelineer

```suggestion
                  required python:view.required and 'required' or None;
```

and remove the `tal:define` line above.